Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -540,6 +541,7 @@ If a user configures a limit or request value that is too low, Rook will still r
 * `crashcollector`: 60MB
 * `mgr-sidecar`: 100MB limit, 40MB requests
 * `prepareosd`: no limits (see the note)
+* `exporter`: 128MB
```

Comment:
@avanthakkar Can you add your findings here for the memory/cpu requirements? In our offline conversation, sounds like it was much lower than this.

Reply:
@avanthakkar do you think 128MB is too much or okay for the exporter on each node?